### PR TITLE
Added `avn events, credits, cards` commands

### PR DESCRIPTION
**Pull request by ftisiot**
# What changed, and why it matters

Added docs about `avn` cli events, credits and cards

### CODE REVIEW

File: docs/tools/cli/card.rst [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  * - ``--number``
+    - Credit card number
+  * - ``--update-project``
+    - Assign card to a project
+
+**Example:** Add a credit card to a named project.
+
+::
+
+  avn card add --cvc 000         \
```
Comment (hab-anka) [inline docs/tools/cli/card.rst:41]: I wonder if we should not use example values in the examples, such as name, card number. We can maybe agree on the standard values we can use for that?
A generic question - same goes for other examples here, and in other CLI commands.

**Review by hab-anka — APPROVED** [review]
General comment about example values - should we make them more realistic (using realistic names, numbers, etc.)?

File: docs/tools/cli/events.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+List project events
+-------------------
+
+Commands for listing events related to a particular project.
+
+
+``avn events``
+'''''''''''''''''''''''
+
+Lists instance or integration creation, deletion or modification events.
```
Comment (hab-anka) [inline docs/tools/cli/events.rst:16]: Is this like a log of events? It is not exactly clear what the events are? Should we maybe show more realistic info in the example?

**Review by hab-anka — APPROVED** [review]
For the events, I wonder if we can describe it better - what kind of events are meant here? Is it like "log" information for the project?

File: docs/tools/cli/card.rst (hunk 1 above)
Comment (lornajane) [reply docs/tools/cli/card.rst]: A good question, I updated some of the examples.

File: docs/tools/cli/events.rst (hunk 2 above)
Comment (lornajane) [reply docs/tools/cli/events.rst:36]: I had to look into this to answer the question and it turns out these are not logs (as I had thought) but more like an audit trail of operations performed on the project. I've updated this page to give more information and an output example

**Comment by lornajane** [comment]
I had to do so much research to answer my own questions that I ended up editing the pages rather than commenting on them all! @ftisiot , @hab-anka any comments?

**Pull request merged**